crossdock/common/constants.go

```diff
@@ -30,6 +30,12 @@ const (
 	// DefaultServerPortTChannel is the port where TChannel server runs
 	DefaultServerPortTChannel = "8082"
 
+	// DefaultEndToEndServerPort is the port where end to end HTTP server runs
+	DefaultEndToEndServerPort = "8083"
```

Comment:
ok, from the design doc looks like you agree to share the same container. Then why run a different port?  We already have an http server on 8081 that responds to queries from the test driver for specific behaviors (the inter-client "trace" behavior). I think you should simple add another endpoint to that server.